utils/net_utils/Cargo.toml

```diff
@@ -6,11 +6,11 @@ edition = "2021"
 
 [dependencies]
 anyhow           = "1.0"
-async-std        = { version = "1.9", features = ["attributes"] }
+async-std        = { version = "1.12", features = ["attributes"] }
 futures          = "0.3"
-isahc            = "1.6"
+isahc            = "1.7"
 log              = "0.4"
 pbr              = "1.0"
 pin-project-lite = "0.2"
 thiserror        = "1.0"
-url              = "2.1"
+url              = "2.3"
```

Comment:
Just FYI if you didn't see other message, updating the dependency from `2.1` to `2.3` is not a particularly important thing (or beneficial, even). The `2.1` version bound means greater than `2.1` and less than `3.0`. We're already using version `2.3.1` of `url` and forbidding older versions isn't all that useful.

The purpose of the original issue was to bump the major versions of our dependencies where possible.

Reply:
If I update it everywhere what issue could it cause?

Reply:
It should be fine but it doesn't provide any benefit.